Assets/Scripts/Importing/GTX/GTX.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Text;
+using SanAndreasUnity.Importing.Archive;
+using SanAndreasUnity.Utilities;
+using UnityEngine;
+
+namespace SanAndreasUnity.Importing.GXT
+{
+    public class GTX : IDisposable
+    {
+
+	    #region LoadPath
+		//this whole region needs refactoring
+	    enum Language

```

Comment:
it may be better to enumerate files in "text" directory, but it's ok

Reply:
Check #24 
It may be better because someone may not have a certain language gxt file? Do I understand you right?